grafana-sync.py

```diff
@@ -333,6 +328,78 @@ def auth_keys(base_url, session):
     request.raise_for_status()
     print(request.json())
 
+class GrafanaLogin:
+    def __init__(self, token:str, user:str, password: str, base_url: str):
+        self.session = requests.Session()
+        if token:
+            self.session.headers = {"Authorization": f"Bearer {token}"}
+        else:
+            create_token(base_url, self.session, user=user, password=password)
+class GrafanaClient:
```

Comment:
please leave space between classes
```suggestion
            create_token(base_url, self.session, user=user, password=password)

class GrafanaClient:
```